site/src/sphinx/setup-configuration.rst

```diff
@@ -268,3 +275,81 @@ Once you have an access to a ZooKeeper cluster, update the ``replication`` secti
 
   -  the minimum allowed age of log items before they are removed from ZooKeeper. If ``null`` the default
      value of '3600000 milliseconds' (1 hour) is used.
+
+Configuring TLS
+---------------
+Central Dogma supports TLS for its API and web pages. To enable TLS, a user may configure ``tls`` property
+in ``dogma.json`` as follows. Note that Central Dogma does not support Server Name Indication(SNI) extension
+at the moment.
```

Comment:
- How about adding a space before `(`?
- Would there any use case of SNI for Central Dogma ?

Reply:
> Would there any use case of SNI for Central Dogma ?

I think that there's not so much use case of SNI. I guess most of our users may run their Central Dogma under a single service domain. Would it be better to remove the sentence?